scripts/visum.sh

```diff
@@ -1,14 +1,43 @@
-if [ "$1" == "" ]; then
-  echo "No file path provided."
+print_error() {
+  >&2 echo $@
+}
+
+if [[ "$1" == ""  ]]; then
+  print_error "No file path provided."
+  exit 1
+fi
+if [ ! -f "$1" ]; then
+  print_error "$1: No such file or directory"
+  exit 1
+fi
+if [ ! -r "$1" ]; then
+  print_error "$1: Permission denied"
   exit 1
 fi
 
+is_office_file() {
+	file_format=`file $1 | rev | cut -d':' -f1 | rev`
+	supported_formats=( word excel composite microsoft powerpoint )
+
+	# verify file format is one of supported_formats
+	echo "${file_format,,}" | grep -qE "\b(`local IFS="|"; echo "${supported_formats[*]}"`)\b"
+	echo $?
+}
+
+FILEPATH=`realpath $1`
+if [ "$(is_office_file $FILEPATH)" == 1 ]; then
+	print_error "$1: Invalid file format"
+	exit 1
+fi
```

Comment:
Could you be consistent with your indentation?

Reply:
Done! Sorry I didn't see that at first :)